EmbeddedSocialClient/sdk/src/main/res/layout/es_view_enter_note.xml

```diff
@@ -31,14 +31,15 @@
 			android:layout_height="wrap_content"
 			android:layout_marginLeft="16dp"
 			android:inputType="textMultiLine|textCapSentences"
-		/>
+			/>
 		<ImageButton
 			android:id="@+id/es_attachImageButton"
 			android:layout_width="wrap_content"
 			android:layout_height="wrap_content"
-			android:src="@drawable/ic_image_black_36dp"
+			android:src="@drawable/es_ic_add_image"
 			android:background="?attr/es_buttonBackground"
-			/>
+            android:tint="@color/es_button_base_gray"
+        />
```

Comment:
I see inconsistent placement of /> w.r.t. the opening < in this file. Perhaps GitHub is not showing it correctly or there are tabs in the file. Can you please check?